### PR TITLE
[XrdCrypto,XrdSecgsi] Update min/default RSA bits to 2048

### PR DESCRIPTION
**Pull request by amadio**

### CODE REVIEW

File: src/XrdSecgsi/XrdSecProtocolgsi.cc [hunk 1]
```diff
@@ -149,7 +149,7 @@ String XrdSecProtocolgsi::UsrCert  = "/.globus/usercert.pem";
 String XrdSecProtocolgsi::UsrKey   = "/.globus/userkey.pem";
 String XrdSecProtocolgsi::PxyValid = "12:00";
 int    XrdSecProtocolgsi::DepLength= 0;
-int    XrdSecProtocolgsi::DefBits  = 512;
+int    XrdSecProtocolgsi::DefBits  = 2048;
```
Comment (abh3) [inline src/XrdSecgsi/XrdSecProtocolgsi.cc:152]: This really should come from the default value define in CryptoAux which is already included in this cc file.

File: src/XrdSecgsi/XrdSecProtocolgsi.hh [hunk 2]
```diff
@@ -188,7 +188,7 @@ public:
    char  *proxy;  // [c] user proxy  [/tmp/x509up_u<uid>]
    char  *valid;  // [c] proxy validity  [12:00]
    int    deplen; // [c] depth of signature path for proxies [0] 
-   int    bits;   // [c] bits in PKI for proxies [512] 
+   int    bits;   // [c] bits in PKI for proxies [2048]
```
Comment (abh3) [inline src/XrdSecgsi/XrdSecProtocolgsi.hh:191]: Same here, refer to the CryptoAux definition so that when it becomes 4096 we know where to look.

**Review by abh3 — CHANGES_REQUESTED** [review]
Just two changes that I saw and one is a comment.  Nice job though!

**Comment by amadio** [comment]
@abh3 I hope the current version is in line with what you suggested.

**Review by abh3 — APPROVED** [review]

**Comment by abh3** [comment]
Thanks! Now we can simply change the define symbol in the future. I guess we need to wait for @esindril to complete the review.

**Review by esindril — APPROVED** [review]
Looks goog!

**Comment by amadio** [comment]
Thanks, @esindril, I'm merging it then. Cheers,

**Pull request merged**